toptal/picasso
# feat(Helpbox): add "as" property to Helpbox.Content component

re #789
### Description

`Helpbox.Content` component is rendered as element `<p>` by default. The new property allows to change it to another tag, like `<div>` to support block content.

### How to test

- See the last example at https://picasso.toptal.net/add-as-prop-to-helpbox-content/?path=/story/components-folder--helpbox

### Review
